src/pages/dashboardTab/components/tabController.js

```diff
@@ -224,7 +224,7 @@ const tabController = (classes) => {
   const TABContainers = tabContainers.map((container) => (
     <TabContainer id={container.id}>
       <TabView
-        options={getOptions(container, classes)}
+        options={getOptions(container, classes, undefined, undefined)}
```

Comment:
Can we add the default values in bento-tools for this function. So we don't need to add undefined in here